src/pages/cli/start/workflows.mdx

```diff
@@ -58,6 +58,20 @@ Once you have made your category updates, run the command `amplify push` to upda
 The `amplify pull` command operates similar to a _git pull_, fetching upstream backend environment definition changes from the cloud* and update the local environment to match that definition. The command is particularly helpful in team scenarios when multiple team members are editing the same backend, pulling a backend into a new project, or when connecting to [multiple frontend projects](/cli/teams/multi-frontend) that share the same Amplify backend environment.
 > \* The CLI will use the S3 deployment bucket to retrieve your project information including the CloudFormation templates from the last successful `amplify push`.
 
+<Callout> 
+
+To generate `aws-exports.js` in an existing project (or checkout a back-end environment that exists in the console but does not show up locally):
```

Comment:
```suggestion
To generate client configuration files (`aws-exports.js`, `amplifyconfiguration.json`, or `amplifyconfiguration.dart`)  for an existing project that is listed in the Amplify Hosting console:
```

Reply:
My only concern here is that we're not also calling out how to checkout a specific environment, if the user has more than one. How about:

```
To generate client configuration files (`aws-exports.js`, `amplifyconfiguration.json`, or `amplifyconfiguration.dart`) for an existing project that is listed in the Amplify Hosting console, or to checkout a back-end environment that has not been pulled locally:
```